### PR TITLE
Dev 1.0

### PR DESCRIPTION
**Pull request by erllypaguntalan**
User:
- User Login/Logout (OpenID: Facebook)
- View own profile
- View other user's profile
- View tracked threads

Thread:
- Add new thread
- View all topics/threads
- Edit/Delete own thread
- Assigning a topic
- Pagination
- Track/Untrack threads

Comments:
- Add a comment to a thread
- View all comments on a thread
- Edit/Delete own comment
- Like/Unlike comment
- Notifications (low prio)

### CODE REVIEW

**Comment by maragatchalian** [comment]
Please make sure that you're using 4 spaces, and not 4 tabs. Please double check, because some parts of your codes are in 'tab' :)

**Comment by maragatchalian** [comment]
Please follow the proper format of your comments. Refer to [pep8](https://www.python.org/dev/peps/pep-0008/)

File: app/templates/user.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    {% block content %}
+        <table>
+            <tr valign="top">
+                <td><img src="http://graph.facebook.com/{{ user.social_id.split('$')[1] }}/picture?type=normal"></td>
+                <td>
+                    <h1>Name: {{ user.name }}</h1>
+                    <h3>Nickname: {{ user.nickname }}</h3>
+                    <h3>Email: {{ user.email }}</h3>
+                    <!-- <h3>About me: {{ user.about_me }}</h3>
+                    {% if user.id == g.user.id %}<p><a href="">Edit</a></p>{% endif %} -->
```
Comment (maragatchalian) [inline app/templates/user.html]: Lines 12 - 13:
Please remove unused codes. Apply to all parts of your codes as well po. :bow:

File: app/templates/create.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          </tr>
+          <tr>
+              <td>Topic:</td>
+              <td>{{ form.topic }}</td>
+              {% for error in form.topic.errors %}
+              <span style="color: red;">[{{ error }}]</span><br>
+              {% endfor %}
+          </tr>
+          <tr>
+              <td>Body:</td>
```
Comment (maragatchalian) [inline app/templates/create.html]: Line 27: 
Suggestion -- change 'Body' to 'Description' or any other term that would make the user understand that this field describes what the title of the thread is all about. :smile:

e.g.,
Title: Roses
Topic: Music
Description: This album is by The Chainsmokers.

:smile:

File: app/templates/user.html (hunk 1 above)
Comment (erllypaguntalan) [inline app/templates/user.html]: Done po.

File: app/templates/create.html (hunk 2 above)
Comment (erllypaguntalan) [inline app/templates/create.html]: Done po.

**Comment by erllypaguntalan** [comment]
-UPDATE-
Modified Database Model

**Comment by erllypaguntalan** [comment]
-UPDATE-
Fixed timestamp

**Comment by erllypaguntalan** [comment]
-UPDATE-
Create Thread Button Modal

**Comment by erllypaguntalan** [comment]
-UPDATE-
Added requirements.txt file

**Comment by erllypaguntalan** [comment]
-UPDATE-
Changed pagination

**Comment by erllypaguntalan** [comment]
-UPDATE-
Fixed Input Validation

**Comment by josefmonje** [comment]
good job. some more practice then study django :)

**Comment by maragatchalian** [comment]
Why was this reverted? :smiley:

**Comment by erllypaguntalan** [comment]
mis-click :sob:

**Pull request merged**